content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst

```diff
@@ -2,99 +2,141 @@
-
-.. image:: create_a_second_warehouse/inventory-adjustment-configuration.png
-   :align: center
-   :alt: A filled out form for an inventory adjustment.
+If a new warehouse is created to represent a real-world storage location with existing stock, update
+the warehouse's stock using an inventory adjustment.
+
+.. seealso::
+   :doc:`count_products`
 
-Once the form is properly configured, click on :guilabel:`Start Inventory` to be taken to the next
-page where products can be added to the inventory adjustment. Add a new product by clicking on
-:guilabel:`Create` and then fill out the product line as follows:
+Inventory can be added to a new warehouse via an inventory adjustment. Navigate to
+:menuselection:`Inventory app --> Operations --> Physical Inventory`, and click :guilabel:`New`.
+Doing so adds a new line at the bottom of the list of inventory adjustments.
 
-- :guilabel:`Product`: the product being added to inventory
-- :guilabel:`Location`: the location where the product is currently stored in the new warehouse;
-  this can be set as the overall warehouse or a location within the warehouse
+Configure the line by filling out the following fields as necessary:
+
+- :guilabel:`Location`: the location where the product is currently stored in the new warehouse.
+  This can be set as the overall warehouse, or a location within the warehouse.
+- :guilabel:`Product`: the product being added to inventory.
 - :guilabel:`Lot/Serial Number`: the lot that the product belongs to or the serial number used to
-  identify it
-- :guilabel:`On Hand`: the total quantity of the product stored in the location for which inventory
-  is being adjusted; this should be zero for a new location or warehouse
-- :guilabel:`Counted`: the amount of the product that is being added to inventory
-- :guilabel:`Difference`: the difference between the *On Hand* and *Counted* values; this will
-  automatically update to reflect the value entered in the :guilabel:`Counted` column
-- :guilabel:`UoM`: the unit of measure used for counting the product
-
-.. image:: create_a_second_warehouse/product-line-configuration.png
+  identify it, if the product is tracked.
+- :guilabel:`On Hand Quantity`: the total quantity of the product stored in the location for which
+  inventory is being adjusted. For a new warehouse or location, this should be set to `0.00`.
+- :guilabel:`UoM`: the unit of measure (UoM) used for counting the product.
+- :guilabel:`Counted Quantity`: the amount of the product being added to inventory.
+- :guilabel:`Difference`: the difference between the on-hand and counted quantities. This
+  automatically updates to reflect the value entered in the :guilabel:`Counted Quantity` column.
+- :guilabel:`Scheduled Date`: the date selected for the next inventory count for this product.
+- :guilabel:`User`: the user who recorded the inventory adjustment in the database.
+
+.. image:: create_a_second_warehouse/create-a-second-warehouse-inventory-adjustment.png
```

Comment:
This screenshot is a bit small and has both x and y axis scrolls visible. Consider retaking up to 933px width so we really capture those details for the reader, as well as at a good crop. It would also be nice to see the column header labels so we know what we're looking at.